### DISCUSSION TITLE
Possibility of implementing CustomSort akin to CustomFilter

### DISCUSSION THREAD

**Discussion by wotgh** [abbasudo/laravel-purity#59] [Ideas]
Hello there.

First of all, thanks for this package, I had previously been using spatie/QueryBuilder for a project, but found it did not suit my needs too well, was directed to Purity and it seems to work like a charm. 

Unfortunately I see Purity does not have a CustomSort functionality yet. Does anyone know if this is something that is in the works or if there are plans on implementing this feature in the future ? :)

Kind regards

**Comment by abbasudo** [comment]
@wotgh Hello there!

Thanks for choosing Purity over other packages, and I'm glad to hear it's working well for you! 

Regarding your question about CustomSort functionality, it's not currently a feature of Purity. However, we're always open to enhancing the package based on the needs of our community. Also maintaining extensibility (and simplicity) is indeed valued in Purity. If you'd like to contribute this feature, we'd greatly appreciate it! Thanks again for your support!

**Comment by iamafasha** [comment]
Any attempts to work on this? I have an interesting use case where I want to be able to sort with two fields(longitude and latitude) to search for items near a given location.

**Reply by abbasudo** [reply]
This feature is planned for later.  For now you can sort by two fields by `GET /api/items?sort[0]=longitude&sort[1]=latitude`
